html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Application.pm

```diff
@@ -379,10 +377,10 @@ sub _render {
         return $self->i18n(@_);
     };
     $args->{ i18n_format } = sub {
-        return $self->i18n_format(@_);  
+        return $self->i18n_format(@_);
     };
 
-    get_logger->info("previous : ".$self->previous_module_id.", current : ".$self->current_module_id);
+    get_logger->info("previous : ". $self->previous_module_id // "undef" .", current : ". $self->current_module_id // "undef");
```

Comment:
This change makes it that only `previous : ` display when previous_module_id is undef and it doesn't display the rest of the line